llvm/test/CodeGen/DirectX/finalize_linkage.ll

```diff
@@ -0,0 +1,60 @@
+; RUN: opt -S -dxil-finalize-linkage -mtriple=dxil-unknown-shadermodel6.5-compute %s | FileCheck %s

```

Comment:
Can you add a run line that uses llc to run the full pipeline and verifies that the `hlsl.export` string attribute gets removed?

Reply:
Done ;)